crates/masonry/src/widget/checkbox.rs

```diff
@@ -4,13 +4,12 @@
 //! A checkbox widget.
 
 use accesskit::{DefaultActionVerb, Role, Toggled};
-use kurbo::{Affine, Stroke};
 use smallvec::{smallvec, SmallVec};
 use tracing::{trace, trace_span, Span};
+use vello::kurbo::{Affine, BezPath, Cap, Join, Size, Stroke};
```

Comment:
I think I'd prefer these all to use `crate::{...}`

If we need the types, we should export them directly, probably.